### ISSUE TITLE
New Logo Design

### ISSUE THREAD

**Issue by Yasujizr** [goxr3plus/XR3Player#6]
![xr3player](https://user-images.githubusercontent.com/36993664/38274393-5861e854-3797-11e8-8b0d-f16184cd6cc9.jpg)

Hi,

I have studied your work and I think a new design will make you more visible. I designed for you, I hope you like it and you want to use it. If you want to reach me, you can look at the my profile. I can give you all the formats of the design free. If you want a change please specify.

**Comment by goxr3plus** [comment]
O MY GOD ♥ ♥ ♥ ♥ ♥ ♥ ♥ ♥ ♥ ♥ ♥ ♥ ♥ ♥ ♥ ♥ ♥ Your design is awesooooome i love it.

**Comment by goxr3plus** [comment]
I like your first icon more because a transparent one doesn't deal well with different backgrounds colors. I want us to discuss a little bit the color of the logo.

I saw your profile and the way your are promoting free logos for other projects, that's amazing.

Sure let's keep in touch. I followed you :)

**Comment by Yasujizr** [comment]
I am glad you like it. Thank you. :)

Is there a color you especially like? According to him, I will regulate the colors again.

Here are a few examples;

![xr3playercolor](https://user-images.githubusercontent.com/36993664/38307675-29ccb4a2-381d-11e8-9c81-84c361e1718d.jpg)

**Comment by goxr3plus** [comment]
Damn black and cyan are also gorgeous. Bro you do fantastic work. I would like to have you as the designer of UI for XR3Player. I wish i could pay you if when it  becomes commercial.

**Comment by goxr3plus** [comment]
I will post a video on YouTube and let the users decide and mention your name.

**Comment by Yasujizr** [comment]
![xr3player3](https://user-images.githubusercontent.com/36993664/38313510-e4770028-382c-11e8-9d91-533bb0d94bc0.jpg)

:) I'm honored. Thank you. I wanted to do UI, but unfortunately I'm not good at this. If I can improve myself in the future, we can work together. :) I do not recommend the diamond because it has too much detail, but this is your logo. I did it like the top to keep the integrity from breaking. I hope you like it. :)

**Comment by goxr3plus** [comment]
Can you make the play button look like a diamond that fell on side? One guy just recommended me. I think this would be the perfect final logo.

I will recommend you so books for graphics designing that i also read. 

One is called '' 100 Things Every Designer should know about the people ''


Just to see. :)

**Comment by goxr3plus** [comment]
@Yasujizr You can be excellent at anything!!! :)

**Comment by Yasujizr** [comment]
![xr3player6](https://user-images.githubusercontent.com/36993664/38324301-8fec8b74-3848-11e8-8641-86c2995a7e81.jpg)


Thanks for your thoughts and advice. :) Good idea, I made an example like this.

**Comment by goxr3plus** [comment]
Great amazing until now, i will choose one of them :). Do you have Discord account or any other so we can speak and share ideas.

'' Follow Excellence and success will come to you ''

**Comment by goxr3plus** [comment]
By the way in which program you are making the logos? Adobe Photoshop or something like paint.net :)?

**Comment by Yasujizr** [comment]
Of course, I sent the Discord link as Mail. I am using Adobe Illustrator CC as the program.

**Comment by goxr3plus** [comment]
@Yasujizr I am still thinking seriously which one to choose :). Meanwhile i am preparing the next update.

I received your email with Discord account i will add you :)

Hey i inform you very soon, it's a hard decision :)

**Comment by Yasujizr** [comment]
Okey, I'm looking forward to. Do not forget to add me to the discord. :)

**Comment by goxr3plus** [comment]
@Yasujizr  My dear friend i love the below logo 
![screenshot24251](https://user-images.githubusercontent.com/20374208/38456931-ae915bd0-3a92-11e8-8fef-d5bbda95de1b.png)

Can i have sizes ( 32 x 32 , 64 x 64 , 128 x 128 ) both .png and .ico extensions . I will add your name and profile on Read Me of repository as the logo designer .

**Comment by goxr3plus** [comment]
Thank you so much , i have a small question though , the name of the file says dimension is **Logo64** but it is 48x48 and the other says **Logo33** and it is 138x138 and the other says **Logo128** and it is 48x48 , am i wrong or it is some kind of technique to keep the quality :) . I will add them right now on the next XR3Player release which guess will be today my boy!!!!!!!!!!!!!!!!!!!!!

![images](https://user-images.githubusercontent.com/20374208/38457892-0bb347d6-3a9f-11e8-91b0-a26c1b8ec9c3.png)

**Comment by goxr3plus** [comment]
For example these are the current icons i have which are looking very sharp and great . They are 64 pixels * 64 pixels .

I have this .png which i converted to .ico also
![icon](https://user-images.githubusercontent.com/20374208/38458078-45b6cb44-3aa2-11e8-9889-91a6224af514.png)

**Comment by Yasujizr** [comment]
You're right, somewhere over the wrong I did I think. I'm work on it now. I will send it again when it's over. I am sorry for these problems.

**Comment by Yasujizr** [comment]
https://drive.google.com/open?id=1Z53S-wS9E78dzlmCzYg_RTYQrsRBg22s

I threw new .ico files. Would you test it?

**Comment by goxr3plus** [comment]
@Yasujizr My friend keep it opened. I will change the logo, though it is not the most important thing this moment. I remember you don't worry ♥ ♥ ♥ ♥

**Comment by goxr3plus** [comment]
Hello :) Can we work on the logo again :) 

![chrome_2018-10-24_16-41-39](https://user-images.githubusercontent.com/20374208/47434733-d653dd80-d7ab-11e8-901c-2789047a828e.png)

I love this one

**Comment by goxr3plus** [comment]
With bigger letters :)

**Comment by HelgeStenstrom** [comment]
I only intended to unassign myself. I know nothing about logo design.

**Comment by goxr3plus** [comment]
Just added you to give opinions :) I'll create a new logo, i do graphic design too, i have designed the current logo of XR3Player and everything other.
